cpp/unreal_plugins/SpCore/Source/SpCore/ArrayDesc.h

```diff
@@ -10,7 +10,7 @@
 #include <string>
```

Comment:
Let's align the header include comments in this file.

Reply:
The current alignment is consistent with the convention described above, since we put C headers in their own block, and the comments are aligned within each block. Do you disagree with the conventions?

Reply:
Going with the convention mentioned [here] (https://github.com/isl-org/spear/pull/264#discussion_r1502140177), this is correct.

Reply:
Confirmed.